kod/object/item.kod

```diff
@@ -1253,6 +1253,23 @@ messages:
       return bDone;
    }
 
+   HideHiddenAttributes()
+   {
+      local i, bDone;
+
+      bDone = FALSE;
+      for i in plItem_Attributes
+      {
+         if NOT Send(self,@GetIDStatusFromCompound,#compound=First(i))
+         {
+            setNth(i,1,(First(i)-1));
+            bDone = TRUE;
+         }
+      }
+      
+      return bDone;
```

Comment:
This return value is not documented (neither is the function), and appears to have no effect.

Reply:
Not that this is valid documentation (and I’ll remove it), but this function serves the opposite purpose of the `RevealHiddenAttributes` function just above it. I hadn’t considered that the return value might not be necessary here. Thinking it through, the return value seems unnecessary for this function’s purpose and appears to only be passed by the reveal function to handle a few unrelated tasks.